taskcluster/ci/ui-test-apk/kind.yml

```diff
@@ -157,11 +158,11 @@ tasks:
                 GOOGLE_PROJECT: 'moz-focus-android'
     focus-arm-beta:
         attributes:
-            build-type: nightly-firebase
+            build-type: focus-beta-firebase
```

Comment:
I'm not sure why this was previously `nightly` for a `beta` job. Not sure if that made any difference. Switching to `focus-beta-firebase`

Reply:
Sounds good! I remember stumbling upon it and being surprised too. Let's make this change 👍